packages/ra-ui-materialui/src/input/AutocompleteInput.js

```diff
@@ -281,22 +299,13 @@ export class AutocompleteInput extends React.Component {
                         isRequired={isRequired}
                     />
                 }
-                value={value}
-                onChange={onChange}
-                autoFocus={autoFocus}
-                margin="normal"
-                className={classnames(classes.root, className)}
-                inputRef={ref}
                 error={!!(touched && error)}
                 helperText={(touched && error) || helperText}
+                className={classnames(classes.root, className)}
                 {...options}
-                InputProps={{
-                    classes: {
-                        input: classes.input,
```

Comment:
You're breaking existing code by removing that

Reply:
The `classes` prop is still passed through, but `classes.input` doesn't make sense (there's no `classes.input` in the styles of `AutocompleteInput`). Not sure why it was there in the first place.